ui/apps/platform/src/Containers/SystemHealth/UsageStatistics/UsageStatisticsForm.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+    downloadProductUsageCsv,
+    fetchCurrentProductUsage,
+    fetchMaxCurrentUsage,
+} from '../../../services/ProductUsageService';
+import { getAxiosErrorMessage } from '../../../utils/responseErrorUtils';
+
+function UsageStatisticsForm(): ReactElement {
+    const initialStartDate = new Date();
+    initialStartDate.setDate(initialStartDate.getDate() - 30);
+    const [startDate, setStartDate] = useState(initialStartDate);
+    const [endDate, setEndDate] = useState(new Date());
+    const [currentUsage, setCurrentUsage] = useState({
+        numNodes: 0,
+        numCpuUnits: 0,
+    } as SecuredUnitsUsage);
+    const [maxUsage, setMaxUsage] = useState({
+        maxNodes: 0,
+        maxNodesAt: '-',
+        maxCpuUnits: 0,
+        maxCpuUnitsAt: '-',
+    } as MaxSecuredUnitsUsageResponse);
+    const [errorFetchingCurrent, setErrorFetchingCurrent] = useState<string>('');
+    const [errorFetchingMax, setErrorFetchingMax] = useState<string>('');
+
+    useEffect(() => {
+        fetchCurrentProductUsage()
+            .then((usage) => {
+                setCurrentUsage(usage.data);
+                setErrorFetchingCurrent('');
+            })
+            .catch((error) => {
+                setErrorFetchingCurrent(getAxiosErrorMessage(error));
+            });
+    }, []);
+    useEffect(() => {
+        fetchMaxCurrentUsage({ from: startDate.toISOString(), to: endDate.toISOString() })
+            .then((usage) => {
+                setMaxUsage(usage);
+                setErrorFetchingMax('');
+            })
+            .catch((error) => {
+                setErrorFetchingMax(getAxiosErrorMessage(error));
+            });
+    }, [startDate, endDate]);
+
+    return (
+        <>
+            {errorFetchingCurrent && (
+                <Alert isInline title={errorFetchingCurrent} variant="danger" />
+            )}
+            {errorFetchingMax && <Alert isInline title={errorFetchingMax} variant="danger" />}
+            <div className="pf-u-font-size-lg">Currently secured</div>
+            <Divider style={{ padding: '5px 0px 10px 0px' }} />
+            <DescriptionList
+                columnModifier={{
+                    default: '2Col',
+                }}
+            >
+                <DescriptionListGroup>
+                    <DescriptionListTerm>CPU units</DescriptionListTerm>
+                    <DescriptionListDescription>
+                        {currentUsage.numCpuUnits}
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>Nodes count</DescriptionListTerm>
+                    <DescriptionListDescription>{currentUsage.numNodes}</DescriptionListDescription>
+                </DescriptionListGroup>
+            </DescriptionList>
+
+            <div className="pf-u-font-size-lg" style={{ padding: '20px 0px 0px 0px' }}>
+                Maximum secured
+            </div>
```

Comment:
Accessibility: this should be the appropriate heading level

Reply:
@vjwilson could you reference the accessibility guideline that determines heading levels? I couldn't find one, unfortunately

Reply:
> @vjwilson could you reference the accessibility guideline that determines heading levels? I couldn't find one, unfortunately

PatternFly's docs have a good presentation, starting on this page,
https://v4-archive.patternfly.org/v4/accessibility/accessibility-fundamentals

But the particular point about headings is under the Headings guideline of this interior page,
https://v4-archive.patternfly.org/v4/accessibility/product-development-guide#what-products-should-address

> Heading text should be descriptive. **Correct heading levels should be used to communicate the outline and structure of the page. A common practice is to use a single h1 for the primary headline or logo on a page, h2s to designate major sections, and h3's in supporting subsections**
(I added the emphasis)

Basically boils down to 2 heuristics
- headings should be marked up with h1 | h2 | h3 etc. (divs and spans and visual styling don't count)
- don't skip heading levels (example, don't have h2 then h4, with no h1 or h3 anywhere, or worse h3 then h1 nested beneath it)

For this modal, it may be appropriate for the modal title to be h2, since it is its own "section" on this page, that just happens to pop up when triggered.
Then individual sections within the modal would have heading starting at h3, and sub-sections within each h3 section with have h4, and so on.

cc: @pedrottimark

Reply:
Thanks! Should I keep `className="pf-u-font-size-lg"` if I use `h2`/`h1`?

So far I used `h2` and without `className="pf-u-font-size-lg"` it looks pretty small(see "Currently secured" down below)

<img width="1339" alt="Screenshot 2023-08-21 at 19 46 30" src="https://github.com/stackrox/stackrox/assets/78353299/93fbcefc-b962-49bc-9290-73c1fee788a7">

Reply:
There are still style rules from old StackRox style that may cause quirky behavior of fonts sizes (CSS is global scoped.)

You can use the PatternFly <Title> Component, 
https://v4-archive.patternfly.org/v4/components/title,
to give you an auto-complete friendly way of specifying the h-level and the size for each heading.

Reply:
That worked, thanks!